### PR TITLE
1 - show rhythms on map - Christina

### PR DESCRIPTION
**Pull request by christina-mifsud**

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -23,15 +25,34 @@ export function Home() {
 			});
 	}, []);
 
+	const handleCityClick = (cityName) => {
+		alert(`${cityName} clicked!`);
+	};
+
+	const dataList = [
+		{ id: 1, title: "DC Hand Dancing", location: "Washington DC" },
+		{ id: 2, title: "Rumba Guaguanco", location: "Havana, Cuba" },
```
Comment (mferryRV) [inline client/src/pages/Home.js]: Don't forget URLs!

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -23,15 +25,34 @@ export function Home() {
...
-				<img
-					className="logo"
-					data-qa="logo"
-					src={logo}
-					alt="Just the React logo"
-				/>
+				<img className="map" src={map} alt="North America Map" />
+
+				<div className="icons">
+					{dataList.map((dataItem) => (
```
Comment (mferryRV) [inline client/src/pages/Home.js:53]: Excellent, love this. Well done using keys and onClicks with parameters.

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,4 @@ dist/
 node_modules/
 .DS_Store
 .env
+2023-08-17-UnityPlan+RhythmMap-DevBrief.pdf
```
Comment (mferryRV) [inline .gitignore]: Is this needed?

File: client/src/pages/CityIcon.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const CityIcon = ({ cityName, onClick }) => {
+
+	return (
+		<div className={`${cityName.toLowerCase().replace(/[,\s]+/g, "-")}-icon`}>
```
Comment (mferryRV) [inline client/src/pages/CityIcon.js]: This feels brittle - is there a way to use `style` props to pass the city-specific styles?

**Review by mferryRV — COMMENTED** [review]
This works, let's work together on how to make the styles dynamic

**Review by mferryRV — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (christina-mifsud) [reply .gitignore]: Apologies! I put it in the repo folder for easy access. Removed!

File: client/src/pages/Home.js (hunk 1 above)
Comment (christina-mifsud) [reply client/src/pages/Home.js]: Added! Thank you!

File: client/src/pages/CityIcon.js (hunk 4 above)
Comment (christina-mifsud) [reply client/src/pages/CityIcon.js]: Thank you Matthew I will look into it!

File: client/src/pages/CityIcon.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			left: "570px",
+		},
+	};
+
+const CityIcon = ({ cityName, onClick }) => {
+  const cityStyle = cityStyles[cityName.toLowerCase().replace(/[,\s]+/g, "-")]; // to remove any spaces, commas etc. from location data eg. "Havana, Cuba" to match our styling classes above
+
+	return (
+	// applying city styling classes to div
+    <div style={cityStyle}>
```
Comment (mferryRV) [inline client/src/pages/CityIcon.js:20]: 👀 👀 👀 👀 love this

**Pull request merged**